interface/forms/button_ros/new.php

```diff
@@ -0,0 +1,667 @@
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="MSK6" name="MSK6" type="checkbox" value="on" >
+			<label for="MSK6"> <?php echo xlt('Localized Joint Swelling');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="MSK7" name="MSK7" type="checkbox" value="on" >
+			<label for="MSK7"> <?php echo xlt('Localized Joint Stiffness');?></label>
+		</span>
+		</TD>
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
+					<TEXTAREA NAME="MSK_notes" ROWS=3 COLS=25 WRAP=SOFT STYLE="width: 2in; height: 0.21in"></TEXTAREA>
+					</TD>
+	</TR>
+
+	<TR class="alt">
+	<TD>
+	<B><?php echo xlt('NEURO:');?></B>
+	</TD>
+	</TR>
+	<TR>
+	<TD>
+		<span class="checkbox-button">
+			<input id="NEURO1" name="NEURO1" type="checkbox" value="on" >
+			<label for="NEURO1"> <?php echo xlt('Dizziness');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="NEURO2" name="NEURO2" type="checkbox" value="on" >
+			<label for="NEURO2"> <?php echo xlt('Vertigo');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="NEURO3" name="NEURO3" type="checkbox" value="on" >
+			<label for="NEURO3"> <?php echo xlt('Syncope');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="NEURO4" name="NEURO4" type="checkbox" value="on" >
+			<label for="NEURO4"> <?php echo xlt('Confusion');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="NEURO5" name="NEURO5" type="checkbox" value="on" >
+			<label for="NEURO5"> <?php echo xlt('Memory Loss');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="NEURO6" name="NEURO6" type="checkbox" value="on" >
+			<label for="NEURO6"> <?php echo xlt('Speech Disturbance');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="NEURO7" name="NEURO7" type="checkbox" value="on" >
+			<label for="NEURO7"> <?php echo xlt('Limb Weakness');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="NEURO8" name="NEURO8" type="checkbox" value="on" >
+			<label for="NEURO8"> <?php echo xlt('Paralysis');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="NEURO9" name="NEURO9" type="checkbox" value="on" >
+			<label for="NEURO9"> <?php echo xlt('Involuntary Movements');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="NEURO10" name="NEURO10" type="checkbox" value="on" >
+			<label for="NEURO10"> <?php echo xlt('Balance Issues');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="NEURO11" name="NEURO11" type="checkbox" value="on" >
+			<label for="NEURO11"> <?php echo xlt('Tingling');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="NEURO12" name="NEURO12" type="checkbox" value="on" >
+			<label for="NEURO12"> <?php echo xlt('Numbness');?></label>
+		</span>
+		</TD>
+	
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
```

Comment:
I always got shot down for punctuation in the information to be translated it breaks the translation